docs/common/create_new_rn_feature.md

```diff
@@ -0,0 +1,142 @@
+# Create new RN feature from the scratch
+
+Within the `app` folder create new folder dedicated for your package eg. `custom` and then follow the structure below:
+
+```
+.
+├── app/
+│   │── custom/                    - custom application
+│           │── appRegistry/       - place for RootComponent and CustomStandalonePackage
+│           │── navigation/        - place for StackNavigator
+│           │── scenes/            - folder with custom package screens
+│           │── index.js           - file which is exporting CustomStandalonePackage
+│           └── package.json
+```
+
+## appRegistry
+
+### RootComponent
+
+That file is representing the context providers which are passing data through the component tree for your application without having to pass props down manually at every level.
+
+```js
+<Dimensions.Provider dimensions={this.props.dimensions}>
+  <AuthContext.Provider
+    accessToken={this.props.accessToken || null}
+    bookingId={this.props.bookingId || null}
+    simpleToken={this.props.simpleToken || null}
+  >
+    {this.props.children}
+  </AuthContext.Provider>
+</Dimensions.Provider>
+```
+
+_For more details check [Context documentation]('https://reactjs.org/docs/context.html')_
+
+### CustomStandalonePackage
```

Comment:
When I read this:

1. I do not know how the `CustomStandalonePackage` should look like
2. I do not know why do I need it or what is it used for